frontend/src/Components/EventsCard.jsx

```diff
@@ -4,10 +4,10 @@ import axios from 'axios';
 
 const EventsCard = (props) => {
     const { setFeedback, loggedUser, event } = props;
-    const [ volunteersList, setVolunteersList ] = useState([]);
-    const [ loggedVolunteerPartOfEvent, setLoggedVolunteerPartOfEvent ] = useState(false);
-    const [ loggedVolunteerRequestAccepted, setLoggedVolunteerRequestAccepted ] = useState(false);
-    const [ reload, setReload ] = useState(0);
+    const [volunteersList, setVolunteersList] = useState([]);
+    const [loggedVolunteerPartOfEvent, setLoggedVolunteerPartOfEvent] = useState(false);
+    const [loggedVolunteerRequestAccepted, setLoggedVolunteerRequestAccepted] = useState('');
```

Comment:
I know that empty strings are falsy but to be more consistent since we switch it to true, it's better to keep this variable state initially set to false

Reply:
I fixed it back.